Surveys launch announcement (#6819)
PostHog/posthog.com: contents/blog/why-we-made-surveys.md

---
date: 2023-10-19
title: Why we've launched PostHog user surveys
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - joe-martin
featuredImage: ../images/blog/posthog-survey.png
featuredImageType: full
category: Inside PostHog
---

> Today, we’ve announced user surveys are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 

Reviewer: ```suggestion
Today, we’re announcing [user surveys](LINK TO PRODUCT PAGE WHEN LIVE) are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
```

Reviewer: Pricing link should link to Survey pricing page when it's live

Author: ```suggestion
Today, we’ve announced [user surveys](/surveys) are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 
```

Launches like this are always exciting for us, but this one is _especially_ notable because it firmly establishes PostHog as an all-in-one tool for building better products. 

> Why is this important? Because spreading everything out across multiple platforms _sucks_. Replays in HotJar, feature flags in LaunchDarkly, analytics in Amplitude (but not your SQL insights, those go somewhere else), and experiments in Google Optimize? That only lays a foundation for frustration, incompatibility, and inconsistency.

Reviewer: Add a survey example here?

It’s also a pain having to log in and out of new platforms all the time. 

> Imagine instead if you could create surveys triggered by the same events you track in your analytics, or if you could trace a survey response back to a session replay in a single platform. Imagine if you could test changes with an A/B experiment and launch surveys alongside them to collect feedback, then turn that into a feature flag and roll it out without ever switching tools.
> 
> Imagine if you had your qualitative and quantitative data side-by-side, along with everything you need to dissect it and act on your findings. 
> 
> That’s the vision we have for PostHog — an all-in-one platform where you can research users, observe usage, analyze data, test ideas, deploy changes, and measure impact consistently, and easily. No more tool juggling, compatibility wrestling, unclear pricing, or botched integrations; just a single, open-source platform that does it all.

Reviewer: ```suggestion
Imagine creating surveys triggered by the same events you track in your analytics

Imagine tracing a survey response back to session replay in just a few clicks.

Imagine trialling a new feature as an A/B test, collecting feedback on it with surveys, and finally shipping the new feature to everyone, without ever switching tools.

Imagine your qualitative and quantitative data in one place, connected to the tools you need dissect and act on it.

That’s the vision we have for PostHog – an all-in-one platform where you can research users, observe usage, analyze data, test ideas, deploy changes, and measure impact consistently, and easily. 

No more tool juggling, compatibility wrestling, unclear pricing, or botched integrations – just a single, open-source platform that does it all.
```

Reviewer: I think it's easier to parse + more impactful if we're liberal with the line breaks.

> Surveys aren’t [our defining feature](/handbook/company/values#we-havent-built-our-defining-feature-yet), but they’re an essential part of this vision because they bring qualitative data into PostHog alongside the quantitative and intuitive data we already collected through events and session replays. 

Reviewer: ```suggestion
Surveys aren’t [our defining feature](/handbook/company/values#we-havent-built-our-defining-feature-yet), but they’re an essential part of this vision. They add valuable real user feedback to the powerful, qualitative data you can already analyze using PostHog
```

Don't want to overuse qual + quant, and think we should avoid talking about "collecting data".

> They're _so_ essential that we've heavily prioritized them. Here's a look at the progress since the beta started, and the pace of when new features dropped:
> 
> - **[June 20th:](/changelog/2023#surveys-beta-released)** Beta launched with open text surveys
> - **[July 25th:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types added
> - **[August 8th:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys added
> - **[August 18th:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages added
> - **[September 14th:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation added
> 
> Basically, we’ve made sure that surveys are perfect for the most common use-cases, including [NPS surveys](/tutorials/nps-survey), customer satisfaction surveys, and [product-market fit surveys](/blog/measure-product-market-fit). 

Reviewer: ```suggestion
## What can you do with surveys in PostHog?

Since we shipped the [initial beta in June](/changelog/2023#surveys-beta-released) with open text surveys, we've added:

- **[July 25:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types
- **[August 8:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys
- **[August 18:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages
- **[September 14:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation

These features make surveys perfect for the most common use-cases, including [NPS surveys](/tutorials/nps-survey), customer satisfaction surveys, and [product-market fit surveys](/blog/measure-product-market-fit). 
```

Author: I don't think this suggested header really adds much, so I've tweaked it.

> Dozens of teams participated in the beta and hundreds of developers contributed PRs and ideas to help us get this far. There are over 81,000 engineers in the PostHog community now, and we couldn’t have built surveys without their support and feedback — and we’re thrilled to release surveys fully so they can start collecting feedback from their communities too. 

Reviewer: ```suggestion
Thank you to the dozens of teams that participated in the beta, and the hundreds of developers who contributed PRs and ideas to help us get this far. 

There are over 81,000 engineers in the PostHog community now, and we couldn’t have built surveys without your support and feedback.

We’re thrilled to release surveys fully so they can start collecting feedback from their communities too. 
```

How solid is the 81k figure, really? Feels a bit fluffy / not real. Everyone knows that isn't an "active" figure, right?

Author: We have it literally front and center on the homepage, and we've never seen complaints. I'm not wed to it, but the use of it here feels like the wrong place to challenge it.

---
date: 2023-10-19
title: Why we've launched PostHog user surveys
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author:
  - joe-martin
featuredImage: ../images/blog/posthog-survey.png
featuredImageType: full
category: Inside PostHog
---

Today, we’ve announced [user surveys](/surveys) are out of beta and in general release, complete with [new pricing](/pricing). Short version? Pricing is usage-based, with a free tier for small projects and an improved offer for everyone who participated in the beta. 

Launches like this are always exciting for us, but this one is _especially_ notable because it firmly establishes PostHog as an all-in-one tool for building better products. 

Why is this important? Because spreading everything out across multiple platforms _sucks_. Replays in HotJar, feature flags in LaunchDarkly, surveys in Pendo, analytics in Amplitude (but not your SQL insights, those go somewhere else), and experiments in Google Optimize? That only lays a foundation for frustration, incompatibility, and inconsistency.

It’s also a pain having to log in and out of new platforms all the time. 

> Imagine creating surveys triggered by the same events you track in your analytics

Reviewer 2: ```suggestion
Imagine creating surveys triggered by the same events you track in your analytics.
```

Imagine your qualitative and quantitative data in one place, connected to the tools you need dissect and act on it.

That’s the vision we have for PostHog – an all-in-one platform where you can research users, observe usage, analyze data, test ideas, deploy changes, and measure impact consistently, and easily. 

No more tool juggling, compatibility wrestling, unclear pricing, or botched integrations – just a single, open-source platform that does it all.

Surveys aren’t [our defining feature](/handbook/company/values#we-havent-built-our-defining-feature-yet), but they’re an essential part of this vision. They add valuable real user feedback to the powerful, qualitative data you can already analyze using PostHog

## What's new in PostHog surveys?

Since we shipped the [initial beta in June](/changelog/2023#surveys-beta-released) with open text surveys, we've added:

- **[July 25:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types
- **[August 8:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys
- **[August 18:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages

> - **[September 14:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation

Author: ```suggestion
- **[September 14:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation
- **[October 10:](/changelog/2023#surveys-get-a-makeover)** Redesign and multiple step surveys
- **[October 19:](/changelog/2023#survey-templates-added)** Survey templates for NPS, CSAT, CES and more
```

## What's new in PostHog surveys?

Since we shipped the [initial beta in June](/changelog/2023#surveys-beta-released) with open text surveys, we've added:

- **[July 25:](/changelog/2023#emoji-and-number-surveys-types-released)** Emoji and number rating survey types
- **[August 8:](/changelog/2023#multiple-choice-surveys-released)** Multiple choice surveys
- **[August 18:](/changelog/2023#wait-periods-now-available-for-surveys)** Wait periods and thank you messages
- **[September 14:](/changelog/2023#automatic-nps-scoring-added-to-surveys)** Automatic NPS calculation
- **[October 10:](/changelog/2023#surveys-get-a-makeover)** Redesign and multiple step surveys
- **[October 19:](/changelog/2023#survey-templates-added)** Survey templates for NPS, CSAT, CES and more

These features make surveys perfect for the most common use-cases, including [NPS surveys](/tutorials/nps-survey), customer satisfaction surveys, and [product-market fit surveys](/blog/measure-product-market-fit). 

> Dozens of teams participated in the beta and hundreds of developers contributed PRs and ideas to help us get this far. There are over 81,000 engineers in the PostHog community now, and we couldn’t have built surveys without their support and feedback — and we’re thrilled to release surveys fully so they can start collecting feedback from their communities too. 

Reviewer 3: for posthog in general, not surveys i assume 😄

> Dozens of teams participated in the beta and hundreds of developers contributed PRs and ideas to help us get this far. There are over 81,000 engineers in the PostHog community now, and we couldn’t have built surveys without their support and feedback — and we’re thrilled to release surveys fully so they can start collecting feedback from their communities too. 

Author: ```suggestion
Dozens of teams participated in the beta and hundreds of developers contributed PRs and ideas elsewhere in our code to help us get this far. There are over 81,000 engineers in the PostHog community now, and we couldn’t have built surveys without their support and feedback — we’re thrilled to release surveys fully so they can start collecting feedback from their communities too. 
```
